help/en/html/hardware/dccpp/index.shtml

```diff
@@ -50,20 +50,30 @@
 <h3>Command Stations</h3>
 
     Currently DCC++ Base Stations can be built from either the Arduino Uno
-    or Arduino Mega platforms.  JMRI should support either platform via USB.
+    or Arduino Mega platforms.  JMRI should support either platform via USB or Network connection.
 </div>
```

Comment:
I moved it to the bottom, and added the missing </div> for <div id="mBody>

That should help.  I'm not sure how to validate site pages with the
formatting and all.  I can open the file locally in a browser, but that
isn't really enough.  I probably should have run this through an online
validator at least, though.

- Mark


On Thu, Dec 3, 2015 at 3:40 PM Matthew Harris <notifications@github.com>
wrote:

> In help/en/html/hardware/dccpp/index.shtml
> <https://github.com/JMRI/JMRI/pull/304#discussion_r46608559>:
>
> > @@ -50,20 +50,30 @@
> >  <h3>Command Stations</h3>
> >
> >      Currently DCC++ Base Stations can be built from either the Arduino Uno
> > -    or Arduino Mega platforms.  JMRI should support either platform via USB.
> > +    or Arduino Mega platforms.  JMRI should support either platform via USB or Network connection.
> >  </div>
>
> This should go - it's closing the 'mainContent' section too early causing
> the formatting to be a bit weird.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/JMRI/JMRI/pull/304/files#r46608559>.
>